client/src/components/TrackItem.vue

```diff
@@ -103,7 +104,7 @@ export default defineComponent({
     const style = computed(() => {
       if (props.selected) {
         return {
-          'background-color': `${vuetify.theme.themes.dark.accentBackground}`,
+          'background-color': `${vuetify?.theme.themes.dark.accentBackground}`,
```

Comment:
```suggestion
          'background-color': `${vuetify!.theme.themes.dark.accentBackground}`,
```
?

Or maybe could use an `as` assertion on the return value of `useVuetify()`?

Reply:
changing the `useVuetify()` to not return undefined should negate the optional chaining and null assertion.